www/src/views/starter-library/filtered-starters.js

```diff
@@ -141,7 +141,7 @@ export default class FilteredStarterLibrary extends Component {
               data={Array.from(
                 count(
                   starters.map(
-                    ({ node: starter }) =>
+                    (starter) =>
```

Comment:
can you make sure to run our formatter? Single arguments shouldn't have parens around them per our prettier config.

Reply:
sure, on it.